src/test/java/io/debezium/connector/yugabytedb/YugabyteDBDatatypesTest.java

```diff
@@ -269,6 +277,26 @@ public void testRecordConsumption() throws Exception {
                 }).get();
     }
 
+  @Test
+  public void testSnapshotRecordConsumption() throws Exception {
+    TestHelper.dropAllSchemas();
+    TestHelper.executeDDL("yugabyte_create_tables.ddl");
+    final int recordsCount = 5000;
+    // insert rows in the table t1 with values <some-pk, 'Vaibhav', 'Kushwaha', 30>
+    insertBulkRecords(recordsCount);
+
+    String dbStreamId = TestHelper.getNewDbStreamId("yugabyte", "t1");
+    Configuration.Builder configBuilder = TestHelper.getConfigBuilder("public.t1", dbStreamId);
```

Comment:
I do not see changes to this function and by default this returns a Configuration with snapshot mode set as never. Can you please check if this test works?

Reply:
Update: Fixed the issue in the test.